_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -449,6 +449,8 @@ We recommend ceasing write requests to a cluster before restoring from a snapsho
 
 Snapshots are only forward-compatible by one major version. If you have an old snapshot, you can sometimes restore it into an intermediate cluster, reindex all indexes, take a new snapshot, and repeat until you arrive at your desired version, but you might find it easier to just manually index your data in the new cluster.
 
+Older-version snapshots and the repositories that contain them continue to be compatible with older versions of OpenSearch. If you have a repository containing snapshots taken by an old version, you then upgrade to another version and snapshots are taken after that as well. The old snapshots can be restored to a new cluster running the same version.
```

Comment:
Thanks for the submission @tan3-netapp, are there limitations to using older versioned snapshots in newer versions of OpenSearch? Is it restricted to major versions, for example?

Reply:
I made an update here after encountering this [issue](https://github.com/opensearch-project/OpenSearch/issues/13676). What I updated is just a guarantee that older versioned snapshots can always be restored to a new cluster running the same version even if the snapshot repository has been modified by the new version. (as I put it in the **Question 2** in the [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/13676) mentioned above).

Anyway, I've changed my wording. Please have another look. Thanks!